lighthouse-core/audits/content-width.js

```diff
@@ -38,17 +38,9 @@ class ContentWidth extends Audit {
    * @return {!AuditResult}
    */
   static audit(artifacts) {
-    if (typeof artifacts.ContentWidth === 'undefined' ||
-        typeof artifacts.ContentWidth.scrollWidth === 'undefined' ||
```

Comment:
> I'll remove from this PR

just kidding, couldn't do it. Did c919119c54c0cb5a10a4a5e7c59e53928f2f4e42 instead. Sorry for the PR noise, can remove if you object